### ISSUE TITLE
DSL2: Add MultiVCFAnalyzer

### ISSUE THREAD

**Issue by aidaanva** [nf-core/eager#1069]

**Comment by ilight1542** [comment]
Note: when adding multivcf analyzer ploidy MUST be set at 2 for unified genotyper.

Current default behavior is that any fasta without explicit ploidy will be set at ploidy 2 for the UG execution. 

Please add a check to correctly check that ploidy is not erroneously set at something other than 2 (easiest would be to just add check for params.run_multivcfanalyzer to then force ploidy 2 for UG execution in modules config..., but maybe better to add explicit WARN statement at runtime that ploidy will be changed so that users know what to expect 🤷 ).

**Comment by aidaanva** [comment]
Good point @ilight1542! I will add the necessary checks/enforce the behaviour
